pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1834,7 +1834,12 @@ def buildScriptsAssemble(
       try {
         // This would ideally not be required but it's due to lack of UID mapping in windows containers
         if ( buildConfig.TARGET_OS == 'windows' && buildConfig.DOCKER_IMAGE) {
-            context.bat('chmod -R a+rwX ' + '/cygdrive/c/workspace/openjdk-build*/workspace/build/src/build/*')
+			def cygwin_workspace = context.WORKSPACE
+			if ( !cygwin_workspace.startsWith("/cygdrive") ) {
+				cygwin_workspace = "/cygdrive/c" + cygwin_workspace.substring(2)
```

Comment:
So we force that drive?

Reply:
That's the current behaviour, yes. Will add logic to convert first character to lowercase (when first char is not a forward-slash) and use that.